src/track-promise.h

```diff
@@ -0,0 +1,31 @@
+#ifndef SRC_TRACK_PROMISE_H_
+#define SRC_TRACK_PROMISE_H_
+
+#include "v8.h"
+
+namespace node {
+
+class Environment;
+
+class TrackPromise {
+ public:
+  TrackPromise(v8::Isolate* isolate, v8::Local<v8::Object> object);
```

Comment:
`Isolate`s are always passed around as pointers, even to the `v8` API itself. And handles are basically just fancy pointers, they are always passed around as values. This can be left as-is. :)